.github/actions/spelling/patterns.txt

```diff
@@ -13,3 +13,6 @@ https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0
 ^\[\]\(\){#[a-z-]+}$
 \]\(#[a-z-]+\)
 \]\[[a-z-]+\]
+
+# Inline code
+\`[^\`]+\`
```

Comment:
Minor: perhaps this is safer:
```suggestion
\`[^\` ]+\`
```
? I'm not sure if it will otherwise exclude things in between inline code blocks as well, like
```
long line of `text` wiht mnay tyops `between` inline code
```
I think most inline code is just a single word without spaces?